src/io/output/WritePVD.cpp

```diff
@@ -30,13 +30,13 @@ void WritePVD(const char* FileName, int step,double time){
 	fprintf(fp,"\t<Collection>\n");
 	for(j = 0; j < step+1; j++){
 		//fprintf(fp,"\t\t<Piece Source=\"%s.vtu.%.4d.%.4d\"/>\n", outfileP2,j,i);
-		fprintf(fp,"\t\t<DataSet timestep=\"%d\" file=\"results/%s.%.04d.pvtu\" />\n",j,outfileP2,j);
+		fprintf(fp,"\t\t<DataSet timestep=\"%f\" file=\"results/%s.%.04d.pvtu\" />\n",stepTime[j],outfileP2,j);
```

Comment:
time= instead of timestep= ?

Reply:
As far as I know, `DataSet` tag supports timestep attribute not time attribute.